### DISCUSSION TITLE
Disable x5t attribute in Keycloak JWKS

### DISCUSSION THREAD

**Discussion by Colangelo-Pierre** [keycloak/keycloak#33227] [Q&A]
Hi everyone,

As sha1 is too weak by modern standard, i would like to disable the x5t attribute from my server JWKS but i didn't find a way to do this in the configuration or in the documentation. 

Is this even doable? 

Thank you for your help.

Pierre.

**Comment by jonkoops** [comment]
See #32897

**Reply by Colangelo-Pierre** [reply]
Thank you, so TL;DR : no yet.

**Reply by jonkoops** [reply]
Indeed, this is something we will be evaluating in the future. We do welcome contributions in case you are interested in taking some of this on :)
